test_scripts/Defects/5_0/2380_OnVehicleData_PoliciesManager_must_allow_all_requested_params_in_case_parameters_field_is_omited.lua

```diff
@@ -0,0 +1,259 @@
+    airbagStatus = "VEHICLEDATA_AIRBAGSTATUS",
+    emergencyEvent = "VEHICLEDATA_EMERGENCYEVENT",
+    clusterModeStatus = "VEHICLEDATA_CLUSTERMODESTATUS",
+    myKey = "VEHICLEDATA_MYKEY"
+  }
+
+local responseUiParams = { }
+local requestParams = { }
+local allParams = {
+  requestParams = requestParams,
+  responseUiParams = responseUiParams,
+}
+
+-- [[ Local Functions ]]
+local function pTUpdateFunc(tbl)
+    local VDgroup = {
+        rpcs = {
+            SubscribeVehicleData = {
+                hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },    
+            },
+            OnVehicleData = {
+                hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+            }
+        }
+    }
+    tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = {"Base-4", "NewTestCaseGroup"}
+end
+
+local function setVDRequest()
+    local tmp = {}
+    for k, _ in pairs(VDValues) do
+      tmp[k] = true
+    end
+    return tmp
+  end
+
+  local function setVDResponse()
+    local temp = { }
+    local vehicleDataResultCodeValue = "SUCCESS"
+    for key, value in pairs(VDValues) do
+      local paramName = "clusterModeStatus" == key and "clusterModes" or key
+        temp[paramName] = {
+          resultCode = vehicleDataResultCodeValue,
+          dataType = value
+        }
+    end
+    return temp
+  end 
+
+  local function subscribeVD()
+    requestParams = setVDRequest()
+    local cid = common.getMobileSession():SendRPC("SubscribeVehicleData", requestParams)
+    responseUiParams = setVDResponse()
+    common.getHMIConnection():ExpectRequest("VehicleInfo.SubscribeVehicleData", requestParams)
+    :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", responseUiParams)
+    end)
+    local MobResp = responseUiParams
+    MobResp.success = true
+    MobResp.resultCode = "SUCCESS"
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+    common.getMobileSession():ExpectNotification("OnHashChange")
+  end
+
+  local function sendOnVehicleData(pParam)
+    common.getHMIConnection():SendNotification("VehicleInfo.OnVehicleData", { [pParam] = onVehicleNotification[pParam] })
+    common.getMobileSession():ExpectNotification("OnVehicleData", { [pParam] = onVehicleNotification[pParam] })
+  end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("RAI, PTU", common.policyTableUpdate, { pTUpdateFunc })
+runner.Step("Activate App", common.activateApp)
+
+-- [[ Test ]]
+runner.Title("Test")
+runner.Step("SubscribeVehicleData", subscribeVD)
+for k, _ in pairs(VDValues) do 
+runner.Step("Send OnVehicleData " .. k, sendOnVehicleData, { k })
```

Comment:
perform alignment in for function

Reply:
Add change according comment to pull request: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2005/commits/d91ee0ca221789e3c19d08ced92507b3b95fb0bf